templates/repo/home.tmpl

```diff
@@ -53,7 +53,11 @@
 		{{end}}
 		{{if .Repository.IsArchived}}
 			<div class="ui warning message">
-				{{.locale.Tr "repo.archive.title"}}
+				{{if .Repository.ArchivedUnix.IsZero}}
+					{{.locale.Tr "repo.archive.title_no_date"}}
+				{{else}}
+					{{.locale.Tr "repo.archive.title_date" (DateTime "short" .Repository.ArchivedUnix.FormatLong .Repository.ArchivedUnix.FormatShort) | Safe}}
```

Comment:
```suggestion
					{{.locale.Tr "repo.archive.title_date" (DateTime "long" .Repository.ArchivedUnix.FormatLong .Repository.ArchivedUnix.FormatShort) | Safe}}
```

Show `April` instead of `Apr`